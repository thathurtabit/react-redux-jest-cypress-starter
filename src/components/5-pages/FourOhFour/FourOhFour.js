import React, { Fragment } from "react";
import Logo from "../../1-atoms/Logo/Logo";

const FourOhFour = () => (
  <Fragment>
    <Logo title="React Logo" />
    404
  </Fragment>
);

export default FourOhFour;
