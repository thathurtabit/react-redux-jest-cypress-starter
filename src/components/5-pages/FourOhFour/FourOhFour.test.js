import React from 'react';
import { shallowToJson } from 'enzyme-to-json';
import 'jest-styled-components';
import { shallowWithTheme } from '../../../utils/test_config/testHelpers';

import FourOhFour from './FourOhFour';

describe('FourOhFour', () => {
  it('should render correctly', () => {
    const element = shallowWithTheme(<FourOhFour />);
    expect(shallowToJson(element)).toMatchSnapshot();
  });
});
