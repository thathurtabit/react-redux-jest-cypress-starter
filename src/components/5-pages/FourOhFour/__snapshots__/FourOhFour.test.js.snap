// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FourOhFour should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
